package.json

```
@@ -32,6 +32,13 @@
   "resolutions": {
     "@storybook/react-docgen-typescript-plugin": "1.0.6--canary.9.cd77847.0"
   },
+  "scripts": {
+    "prepare": "husky"
+  },
+  "lint-staged": {
+    "**/*.css": "yarn stylelint --fix",
+    "*.{js,ts,tsx,json,md,yml}": "prettier --write"
```

We should probably add `--ignore-path .eslintignore` to match what `make format` does.

```suggestion
    "*.{js,ts,tsx,json,md,yml}": "prettier --write --ignore-path .eslintignore"
```